sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/util/ClientLogger.java

```diff
@@ -482,8 +368,9 @@ public LoggingEventBuilder atLevel(LogLevel level) {
      */
     @Metadata(conditions = FLUENT)
     public static final class LoggingEventBuilder {
+        private static final JsonStringEncoder JSON_STRING_ENCODER = JsonStringEncoder.getInstance();
         private static final LoggingEventBuilder NOOP = new LoggingEventBuilder(null, null, null, false);
-        private static final String SDK_LOG_MESSAGE_KEY = "sdk.message";
+        private static final String SDK_LOG_MESSAGE_KEY = "{\"sdk.message\":\"";
```

Comment:
maybe this? without `azure` it does not really matter what goes there
```suggestion
        private static final String SDK_LOG_MESSAGE_KEY = "{\"message\":\"";
```

Reply:
Isn't it still important to know its from within the SDK codebase and not a random "message" key?

Reply:
we still have logger category (class name) in the final log record - it's not part of the structured message, but will describe where the log message came from.

The `az.sdk.message` was important to easily json and non-json formatted logs, but now since all of them are json, I think it does not matter.